src/moveFile.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+
+const moveFile = async(src, dest) => {
+  if (!fs.existsSync(src)) {
+    throw new Error(`Source file ${src} doesn't exists.`);
+  }
+
+  const destinationDir = dest.endsWith('/')
+    ? dest
+    : path.dirname(dest);
+
+  if (!fs.existsSync(destinationDir)) {
+    throw new Error(`Destination directory ${destinationDir} doesn't exist.`);
+  }
+
+  const fileName = dest.endsWith('/')
```

Comment:
The condition used to determine the file name could lead to errors. If the destination is a path without a trailing '/', the file will be renamed even if it wasn't the intent.